src/frontend/hooks/useIsFullyFocused.ts

```diff
@@ -30,12 +30,14 @@ export const useIsFullyFocused = () => {
   }, []);
 
   useEffect(() => {
-    let navSubscriptions: (() => void)[] = [];
+    // let navSubscriptions: EventListenerCallback<any, any>  [] = [];
 
-    navSubscriptions = [
+    const navSubscriptions = [
       navigation.addListener("focus", () => setIsNavigationFullyFocused(true)),
       navigation.addListener("blur", () => setIsNavigationFullyFocused(false)),
     ];
+
+    return navSubscriptions.forEach(s => s);
```

Comment:
This is returning an array of functions. As far as I know, `useEffect` expects a single function to be returned, which is called when the dependencies change.

```suggestion
    return () => navSubscriptions.forEach(s => s());
```